moto/appmesh/responses.py

```diff
@@ -97,3 +101,151 @@ def tag_resource(self) -> str:
             tags=tags,
         )
         return json.dumps(dict())
+
+    def describe_virtual_router(self) -> str:
+        mesh_name = self._get_param("meshName")
+        mesh_owner = self._get_param("meshOwner")
```

Comment:
```suggestion
        mesh_owner = self._get_param("meshOwner") or self.current_account
```

The `meshOwner`-parameter is not mandatory, so I assume it defaults to the current account.

There are two ways to handle this, I suppose - either default to the current account, or make sure that the `AppMeshBackend.describe_virtual_router` can deal with an optional `mesh_owner`. It currently always assumes that it is supplied.

I have no preference how it's handled - this is just one suggestion.

I'm pointing it out for this particular method, but as far as I can see, this parameter is optional for a lot of methods - and the current implementation assumes it always exists. So this should be fixed across the board.

Reply:
@bblommers good call, I've made sure owner is optional across the board. Ready for re-review